amplitude.yaml

```diff
@@ -0,0 +1,52 @@
+specVersion: 1.0.0
+name: subquery-nova-karura
```

Comment:
all configs in this PR has name `subquery-nova-karura ` should they be the same?